### DISCUSSION TITLE
How to compile stylus.min.js?

### DISCUSSION THREAD

**Discussion by sotasan** [stylus/stylus#2697] [Q&A] [answered]
Hey, I was wondering how the stylus.min.js is being compiled.
https://stylus-lang.com/try/stylus.min.js

Because I am working on a library for Java and I don't want to include this file in my source code, but build it from source.
https://github.com/shotadotdev/stylus4j
The library is working in it's current state.

Best Regards

**Comment by iChenLei** [comment, marked as answer]
I am new maintainer of stylus, this is also confused for me. so I ask previous maintainer and he say that you can switch to `client` branch and learn how to build browser compatiable stylus. 
> https://github.com/stylus/stylus/pull/2642 My work for it
> https://github.com/stylus/stylus/tree/client client branch

**Reply by sotasan** [reply]
It‘s working, thanks!
